src/lib/fetch-helper.js

```diff
@@ -10,6 +10,16 @@ const checkStatus = async response => {
     // plain text of the response
     try {
       error = JSON.parse(text).error
+
+      // The error is parseable, but null. Inspect the first element of the response
+      // to see if it is a bulk creation element error
+      if (!error) {
+        try {
+          error = JSON.parse(text).data[0].error
+        } catch (e) {
+          error = 'Unknown error: ' + text
+        }
+      }
```

Comment:
Since this is still happening within the initial `try {}`, we should be able to simply do
```suggestion
        error = JSON.parse(text).data[0].error
      }
```